webpack/scenes/ContentViews/__tests__/contentViewTable.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { render, fireEvent, within } from 'react-testing-lib-wrapper';
+import ContentViewsTable from '../Table/ContentViewsTable.js';
+
+const contentViewIndex = require('./contentViewList.fixtures.json');
+
+let firstCV;
+let contentViews;
+beforeEach(() => {
+  const { results } = contentViewIndex;
+  contentViews = results;
+  [firstCV] = results;
+});
+
+const defaultProps = {
+  loadContentViewDetails: () => {},
+  detailsMap: {},
+};
+
+test('Can view content views on the screen', () => {
+  const { getByText, queryByTestId, queryByText } =
+    render(<ContentViewsTable results={contentViews} loading={false} {...defaultProps} />);
+
+  // query* functions will return the element or null if it cannot be found
+  // get* functions will return the element or throw an error if it cannot be found
+  // Loop through the CVs in the fixture and ensure they are present
+  contentViewIndex.results.map(({ name }) => {
+    expect(getByText(name)).toBeTruthy();
+    return name;
+  });
+
+  // Ensure loading text is not showing as a baseline sanity check Using .toBeNull() here as the
+  // loading spinner isn't even in the DOM, as opposed to an element that is in the DOM but hidden.
+  // Also using a test id that is set in the the application code so we can correctly identify the
+  // loading text. Pattern matching by "Loading" seems like it could give false positives, a test
+  // id will ensure we are correctly identifying the loading text.
+  expect(queryByTestId('cv-loading-text')).toBeNull();
+
+  // Ensure empty state text isn't showing as a baseline sanity check
+  expect(queryByText(/You currently don't have any Content Views/)).toBeFalsy();
```

Comment:
Now that we're finding elements by text like users would, I'm curious if we should worry about translations.  Thoughts?

Reply:
I don't see it being an issue in the testing environment as everything defaults to English, but maybe I'm not understanding, do you mean testing with the translated text?

Reply:
@johnpmitsch I just mean will it cause any problems if you run the tests and your system language is set to a non-English language?  And is it something we will need to address?

Reply:
> @johnpmitsch I just mean will it cause any problems if you run the tests and your system language is set to a non-English language? And is it something we will need to address?

I'm not sure, but I would be surprised if we don't make this assumption other places and also we do have steps to standardize our development environment and CI environments so I don't see this being an issue unless there is something I am missing.